modular_newhorizons/master_files/code/modules/background/origins/origins/human/sol.dm

```diff
@@ -0,0 +1,207 @@
+	Venus residents tend to culturally divide between occupational cultures, 'Jintarans' are often quickly identified by the clipped and gruff accent attributed to Venus miners, clad in their high-temperature adapted 'Jintara' Industrial suits."
+	possible_accents = list(ACCENT_VENUSJIN)
+	possible_citizenships = CITIZENSHIPS_SOLARIAN
+	possible_religions = RELIGIONS_SOLARIAN
+
+/singleton/origin_item/origin/mars
+	name = "Mars"
+	desc = "As one of the first solar colonies, Martians as a people have suffered grievously throughout their history. Mars was once dreamed of as the next planet to fully colonize and then turn green, a utopia in the making where humanity could begin its foray into the stars. It was anything but that. Modern Mars consists of rows and rows of low-income housing stuck under habitational bio-domes, more giant slums than anything else. The only orderly domes tend to be the Colony Governor's own, including the Martian Militia dome, as well as any corporate branch office.  As a result many Martians can be found abroad, wanting to escape their dull and dreary life in search of adventure and a better life out there."
+	possible_accents = list(ACCENT_MARTIAN)
+	possible_citizenships = list(CITIZENSHIP_SOL, CITIZENSHIP_BIESEL)
+	possible_religions = RELIGIONS_SOLARIAN
+
+/singleton/origin_item/origin/jupiter
+	name = "Jupiter"
+	desc = "Jupiter as a gas giant lacks a proper habitable 'surface', instead, all of its installations are in orbit around it or on the upper atmosphere of its expanse. While the majority of the orbital facilities are refueling stations or research outposts, all either government or corporate gas refineries, the most prominent is the 'Helios' Habitation Dock, once a simple Protectorate dockyard, now its an Idris owned tourist hotspot, supported by the fact that the habitation dock's residential levels turned into a luxury hotel, Helios hosts many scheduled events along its weekly terran cycles, looping the itinerary every week, which draws in many tourists and celebrities. For the average tourist, they will likely be captivated by the natural wonders of Jupiter such as the Red Spot or the Hexagon, Helios even has telescope theaters, to allow high-resolution public viewings of Jupiters great storms in many different visuals. And for the less visually impressed tourist or even for the permanent hotel residents, Helios is still a top of the line Idris resort, offering many luxury and entertainment options.\
```

Comment:
Something about this text block looks broken.

Reply:
Probably just the fact that I didnt use any real linebreaks for it, fixing.